buildSrc/src/main/java/org/springframework/boot/build/mavenplugin/PluginXmlParser.java

```diff
@@ -65,7 +67,7 @@ private String textAt(String path, Node source) throws XPathExpressionException
 	private List<Mojo> parseMojos(Node plugin) throws XPathExpressionException {
 		List<Mojo> mojos = new ArrayList<>();
 		for (Node mojoNode : nodesAt("//plugin/mojos/mojo", plugin)) {
-			mojos.add(new Mojo(textAt("goal", mojoNode), format(textAt("description", mojoNode)),
+			mojos.add(new Mojo(textAt("goal", mojoNode), format(requireNonNull(textAt("description", mojoNode))),
```

Comment:
I don't think an explicit check is needed.

Reply:
Removed as of commit 976fc62